Dockerfile

```diff
@@ -1,12 +1,17 @@
-FROM quay.io/azavea/scala:2.10.6
+FROM openjdk:7-jre
```

Comment:
Also, should we be using 8 here? That should still be able to run jars compiled with lower versions.